### ISSUE TITLE
[MilestoneEpic] Improve banners

### ISSUE THREAD

**Issue by igor-sirotin** [status-im/status-desktop#12938]
1. When the app is offline, only show the offline banner. All other connections issues banners should be hidden.
I guess there're 2 possible banners to show together with the OFFLINE: `backup seed phrase` and `testnet mode`.

2. Implement a new banner `No connected peers`. 
This should be visible when we have 0 peers, but have connection to the internet. I.e. wallet will work, but messaging won't.

**Comment by igor-sirotin** [comment]
I'd say this is `ui-team`?

**Comment by caybro** [comment]
> I'd say this is `ui-team`?

Yeah I guess so... any more formal description, like link to Figma etc?

**Comment by igor-sirotin** [comment]
> Yeah I guess so... any more formal description, like link to Figma etc?

@caybro nope 🙂 
I wrote this during a call from @John-44 words.

**Comment by caybro** [comment]
Then I guess it's for @benjthayer first at this stage ;)

**Comment by benjthayer** [comment]
Thanks @igor-sirotin @caybro do we want to recommend the user restart the app if they have 0 peers? I'm not sure if that is the best way to resolve the issue but that is what I've been advised to do in the past when this happens?

**Comment by igor-sirotin** [comment]
> Thanks @igor-sirotin @caybro do we want to recommend the user restart the app if they have 0 peers? I'm not sure if that is the best way to resolve the issue but that is what I've been advised to do in the past when this happens?

Theoretically that shouldn't be something that should help. But if you'd ask me, I'd suggest you the same 😄 
Nevertheless, I don't think we should put this recommendation to UI. If we have such a bug that we restart is required to get peers back, then this bug should be addressed. And then such recommendation would be useless.

**Comment by noeliaSD** [comment]
Still some decision to take. Moved due to lack of capacity

**Comment by noeliaSD** [comment]
@benjthayer @xAlisher it should be nice to have your final feedback here! Thanks

**Comment by xAlisher** [comment]
I agree with Johns input. If we show offline banner, all others, instead of **Back up recovery phrase**  and **Testnet mode** must be hidden.
